src/Host/Broker/Impl/Startup/Startup.cs

```diff
@@ -62,7 +64,13 @@ SecurityManager securityManager
             });
 
             app.Use((context, next) => {
-                if (!context.User.Identity.IsAuthenticated) {
+                if (context.Request.Path.Value.Contains("/remoteuri")) {
```

Comment:
Also, can you make it separate from authentication in the pipeline (i.e. two separate `Use` calls, or separate `Use` and `MapRoute`)?